vertx-web-graphql/pom.xml

```diff
@@ -62,6 +64,28 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-junit5</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Can you please get rid of jUnit5 and stick to `vertx-unit` so that we don't mix testing tools?

Reply:
Sure, I've just done it and you can see changes in my last commit, thanks Thomas.